beacon_node/store/src/hot_cold_store.rs

```diff
@@ -2454,7 +2455,7 @@ impl<E: EthSpec, Hot: ItemStore<E>, Cold: ItemStore<E>> HotColdDB<E, Hot, Cold>
         } else {
             AnchorInfo {
                 anchor_slot,
-                oldest_block_slot: anchor_slot,
+                oldest_block_slot,
```

Comment:
Quite a silly bug 🙄 probably I wrote that line before I understood the store well